docs/source/developer/contributing.rst

```diff
@@ -40,17 +49,19 @@ working.  This can be useful to gather feedback, or to stress your changes
 against the :ref:`continuous integration <travis_ci>` platorm.  In this
 case, please prepend ``[WIP]`` to your pull request's title.
 
+.. _buildenv:
+
 Build environment
------------------
+'''''''''''''''''
 
-Numba has a number of dependencies (mostly numpy and llvmpy) with
-non-trivial build instructions.  Unless you want to build those dependencies
-yourself, we recommend you use
-`Conda <http://conda.pydata.org/miniconda.html>`_ to create a dedicated
-development environment and install precompiled versions of those
-dependencies there::
+Numba has a number of dependencies (mostly `Numpy <http://www.numpy.org/>`_
+and `llvmlite <https://github.com/numba/llvmlite>`_) with non-trivial build
+instructions.  Unless you want to build those dependencies yourself, we
+recommend you use `Conda <http://conda.pydata.org/miniconda.html>`_ to
+create a dedicated development environment and install precompiled versions
+of those dependencies there::
 
-   $ <path_to_miniconda>/conda create -n numbaenv python=3.4 llvmpy numpy
+   $ <path_to_miniconda>/conda create -n numbaenv python=3.4 llvmlite numpy
```

Comment:
Should developers be installing llvmlite from the Numba Binstar channel so they get llvmlite master as well?